docs/combininglock.md

```diff
@@ -94,14 +94,14 @@ Let us provide an MCS queue lock for our API given above:
 ```C++
 struct LockNode
 {
-  std::atomic<LockNode*> next{nullptr};
```

Comment:
I would suggest we leave this file untouched as it is for understanding purposes, and the proxy will only serve to complicate the presentation.